index.html

```diff
@@ -7,8 +7,9 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Food Folio</title>
   <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/fd9a149f30.js" crossorigin="anonymous"></script>
```

Comment:
Didn't notice anything from fontawesome being used in the code. Can u pls confirm if it is needed...

Reply:
@ferrary7 Can u pls check

Reply:
Done!

Reply:
The script is still there right?

Reply:
Yes